examples/camunda-spring-boot-starter-example-simple/src/main/java/org/camunda/bpm/spring/boot/starter/example/simple/Showcase.java

```diff
@@ -5,9 +5,9 @@
 import org.camunda.bpm.engine.RuntimeService;
 import org.camunda.bpm.engine.TaskService;
 import org.camunda.bpm.engine.task.Task;
+import org.camunda.bpm.spring.boot.starter.events.ProcessApplicationStartedEvent;
```

Comment:
just do it 👍🏻

> Am 24.07.2016 um 13:10 schrieb Jan Galinski <notifications@github.com>:
> 
> In examples/camunda-spring-boot-starter-example-simple/src/main/java/org/camunda/bpm/spring/boot/starter/example/simple/Showcase.java <https://github.com/camunda/camunda-bpm-spring-boot-starter/pull/115#discussion_r71986744>:
> 
> > @@ -5,9 +5,9 @@
> >  import org.camunda.bpm.engine.RuntimeService;
> >  import org.camunda.bpm.engine.TaskService;
> >  import org.camunda.bpm.engine.task.Task;
> > +import org.camunda.bpm.spring.boot.starter.events.ProcessApplicationStartedEvent;
> We should stick to singular for package names ... so starter.event would be my choice
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/camunda/camunda-bpm-spring-boot-starter/pull/115/files/191bfce06b909def2e5a7cbbe15e5828cb0d83d2#r71986744>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AMTszVkjZsEeZc7sy7RNe4_XVdz0P0Inks5qY0glgaJpZM4JTUhI>.
>